openlibrary/templates/type/edition/book_title.html

```diff
@@ -0,0 +1,17 @@
+$def with (work, edition)
+
+$ book_title = edition.get('title', '') if edition.title else (work.title if work else '')
+
+$if work.series:
+  <h2><a href="/works/$(work.series.work)">($work.series.name)</a></h2>
```

Comment:
Wat?

Reply:
I wonder if this was an experiment that is incomplete. Removing
```suggestion
```